### DISCUSSION TITLE
SUBS command pointing only to China store?

### DISCUSSION THREAD

**Discussion by jbmccune** [chr233/ASFEnhance#125] [提问求助 / Support]
<html>
<body>

SUBS [Bots] <AppIDs\|SubIDs\|BundleIDs> | S | Operator | Get available subs from store page, support APP/SUB/BUNDLE
-- | -- | -- | --

</body>
</html>

This command seems to be pulling from the China site instead of my local US site. For example, if I try "S 990080" (Hogwarts Legacy), I get "App/990080: 霍格沃茨之遗" and if I try "S 1245620" (Elden Ring), I get "App/1245620: Store page not found" - presumably because Elden Ring isn't available in China?

Is there some way I can configure which store this pulls from?

**Comment by chr233** [comment]
i will check the code tomorrow, this function is writen before the plugin support multilingual

**Comment by chr233** [comment]
the Elden Ring's store page has age check dialog, I will fix it.

**Comment by chr233** [comment]
![image](https://github.com/chr233/ASFEnhance/assets/13798576/2962d203-6f32-47c1-9982-7b1827ec5098)

fixed, I will release later
